jbpm-designer-client/src/main/resources/org/jbpm/designer/public/js/Plugins/view.js

```diff
@@ -61,6 +61,9 @@ ORYX.Plugins.View = {
         this.maxZoomLevel = 2.0;
         this.diff=5; //difference between canvas and view port, s.th. like toolbar??
 
+        // source view window
+        this.sourcewin;
+
```

Comment:
@csadilek its not left-over.  just defines sourcewin which is used later in the methods